src/python/gudhi/sklearn/cover_complex.py

```diff
@@ -0,0 +1,473 @@
+        filters (numpy array of shape (num_points) x (num_filters)): filter functions (sometimes called lenses) used to compute the cover. Each column of the numpy array defines a scalar function defined on the input points. Used only if cover = "functional".
+        filter_bnds (numpy array of shape (num_filters) x 2): limits of each filter, of the form [[f_1^min, f_1^max], ..., [f_n^min, f_n^max]]. If one of the values is numpy.nan, it can be computed from the dataset with the fit() method. Used only if cover = "functional".
+        resolutions (numpy array of shape num_filters containing integers): resolution of each filter function, ie number of intervals required to cover each filter image. Must be of length 1 if complex_type = "gic". Used only if cover = "functional". If None, it is estimated from data.
+        gains (numpy array of shape num_filters containing doubles in [0,1]): gain of each filter function, ie overlap percentage of the intervals covering each filter image. Must be of length 1 if complex_type = "gic". Used only if cover = "functional".
+        N (int): subsampling iterations (default 100) for estimating scale and resolutions. Used only if cover = "functional" and clustering or resolutions = None. See http://www.jmlr.org/papers/volume19/17-291/17-291.pdf for details.
+        beta (double): exponent parameter (default 0.) for estimating scale and resolutions. Used only if cover = "functional" and clustering or resolutions = None. See http://www.jmlr.org/papers/volume19/17-291/17-291.pdf for details.
+        C (double): constant parameter (default 10.) for estimating scale and resolutions. Used only if cover = "functional" and clustering or resolutions = None. See http://www.jmlr.org/papers/volume19/17-291/17-291.pdf for details.
+        clustering (class): clustering class (default sklearn.cluster.DBSCAN()). Common clustering classes can be found in the scikit-learn library (such as AgglomerativeClustering for instance). Used only if complex_type = "mapper". If None, it is set to hierarchical clustering, with scale estimated from data.
+        graph (string): type of graph to use for GIC. Used only if complex_type = "gic". Currently accepts "rips" only.
+        rips_threshold (float): Rips parameter. Used only if complex_type = "gic" and graph = "rips".
+        input_name (string): name of dataset. Used when generating plots.
+        cover_name (string): name of cover. Used when generating plots.
+        color_name (string): name of color function. Used when generating plots.
+        verbose (bool): whether to display info while computing.
+ 
+        simplex_tree (gudhi SimplexTree): simplicial complex representing the cover complex computed after calling the fit() method.
+        node_info_ (dictionary): various information associated to the nodes of the cover complex. 
+    """
+    def __init__(self, complex_type="mapper", input_type="point cloud", cover="functional", colors=None, mask=0,
```

Comment:
It feels a bit weird that the `__init__` function is documented in the doc after all the attributes are described (a bit redundant I'd say). 

Note: perhaps the doc I am looking at (https://12448-174304137-gh.circle-artifacts.com/0/sphinx/cover_complex_sklearn_isk_ref.html) is outdated, I don't see the "Initialize self." in this code.

Reply:
Yes, you are right, I put the argument description at the wrong place. I've just fixed the doc.